src/tests/amp/ampEpic.ts

```diff
@@ -1,65 +1,38 @@
+            'And because we believe everyone deserves access to trustworthy news, we keep Guardian reporting open for everyone to read, regardless of where they live or what they can afford to pay.',
+            'Our work is only possible thanks to the support we receive from our readers.',
         ],
+        highlightedText: `If you can, support the Guardian today, from as little as ${currencySymbol}1. Thank you.`,
+        cta: {
+            text: 'Show your support',
+            url: 'https://support.theguardian.com/contribute',
+            campaignCode: campaignCode,
+            componentId: campaignCode,
+        },
     };
 }
 
-async function ampUsEpic(): Promise<AMPEpicResponse> {
-    const campaignCode = 'AMP_USREGION_EPIC';
-    return {
-        items: [
-            {
-                heading: 'A fresh start for America ...',
-                paragraphs: [
-                    '... as Joe Biden and Kamala Harris win the US election. The American people have spoken loudly to disavow the last four years of chaos and division. The voters have chosen a new path. Now, the real work begins.',
-                    'The Guardian welcomes the opportunity to refocus our journalism on the opportunities that lie ahead for America: the opportunity to fix a broken healthcare system, to restore the role of science in government, to repair global alliances, and to address the corrosive racial bias in our schools, criminal justice system and other institutions.',
-                    'Because we believe every one of us deserves equal access to fact-based news and analysis, we’ve decided to keep Guardian journalism free for all readers, regardless of where they live or what they can afford to pay. This is made possible thanks to the support we receive from readers.',
-                ],
-                highlightedText:
-                    'If you can, support the Guardian’s journalism with as little as $1 – it only takes a minute. Thank you.',
-                cta: {
-                    text: 'Support the Guardian',
-                    url: 'https://support.theguardian.com/contribute',
-                    campaignCode: campaignCode,
-                    componentId: campaignCode,
-                },
-            },
-        ],
-    };
-}
-
-export async function ampEpic(countryCode?: string): Promise<AMPEpicResponse> {
-    const ampEpic = await selectAmpEpic(countryCode);
-    if (ampEpic) {
-        return {
-            items: [ampEpic],
-        };
-    }
+export async function ampEpic(
+    ampVariantAssignments: AmpVariantAssignments,
+    countryCode?: string,
+): Promise<AMPEpic> {
+    const ampEpic = await selectAmpEpic(ampVariantAssignments, countryCode);
 
-    // No epic from the tool, fall back on a hardcoded epic
-    return countryCode === 'US' ? ampUsEpic() : ampDefaultEpic(countryCode);
+    // If no epic from the tool, fall back on a hardcoded epic
+    return ampEpic ? ampEpic : ampFallbackEpic(countryCode);
```

Comment:
👍 I think this makes it backwards compatible?
So for the migration we can deploy this dotcom-components change first, and the old AMP pages will continue to get the fallback epic until we have deployed the dotcom-rendering (and pages have de-cached).

Reply:
Yeah, but we still need to update the fallback epic copy.

Reply:
👍 let's make sure we update that first